### ISSUE TITLE
Package RetroArch web player for Chrome Web Store

### ISSUE THREAD

**Issue by matthewbauer** [libretro/RetroArch#6714]
Hi,

Thanks everyone for your work on RetroArch & its cores. You’re helping us preserve our video game history for the future.

I have an old project that uses libretro cores to emulate SNES and NES games. You can check it out here: https://github.com/matthewbauer/gametime-player. I made it before the Emscripten RetroArch had been released and it implements a barebones frontend.

Sadly, I no longer have the time to update my frontend & want to point my users to the RetroArch web player as an alternative. One thing that is missing, though, is a release in the Chrome Web Store. For various reasons this is considered more convenient (especially with chromebooks).

Could you package RetroArch Web into a Chrome web app? I could do it myself but it probably should be done officially.

**Comment by Ferk** [comment]
Google is deprecating web apps, they'll stop working on this quarter: 
https://9to5google.com/2017/12/06/google-chrome-apps-dead-web-store/

I think it's better to adhere to standards and follow things like the [Web App Manifest](https://www.w3.org/TR/2016/WD-appmanifest-20160331/) and other features designed for web apps that would work also across browsers:

**Comment by matthewbauer** [comment]
The web app manifest is definitely useful. But note Google still maintains the store for ChromeOS.

**Comment by MurhafSousli** [comment]
Hi @matthewbauer I am interested to implement an offline web based app that works on mobile phones, can you guide me which emulator did you use in your app?

**Comment by matthewbauer** [comment]
The cool thing about retroarch/libretro is once you get the bindings working you can run almost any emulator that has the libretro bindings. With emscripten, you are probably not going to get anything past 6th gen console working without serious lagging. Snes9x has really good performance just about anywhere:

https://github.com/libretro/snes9x

**Comment by MurhafSousli** [comment]
@matthewbauer The one you have in your app looks very good and the sound is working properly
Is your app using https://github.com/libretro/snes9x?

**Comment by matthewbauer** [comment]
Yes, they're a little old but the new versions should work just as well:

https://github.com/matthewbauer/retrojs/tree/master/core
